src/hotspot/share/runtime/deoptimization.cpp

```diff
@@ -784,6 +784,11 @@ JRT_LEAF(BasicType, Deoptimization::unpack_frames(JavaThread* thread, int exec_m
           // a given bytecode or the state after, so we try both
           if (!Bytecodes::is_invoke(cur_code) && cur_code != Bytecodes::_athrow) {
             // Get expression stack size for the next bytecode
+            if(UseNewCode) {
```

Comment:
I assume you intended to remove this from the final version of the patch.

Reply:
Sorry... I haven't sync up with @chhagedorn, I picked up a starter issue to work on it and missed assignee...